waku/waku_archive/driver/builder.nim

```diff
@@ -92,13 +101,11 @@ proc new*(T: type ArchiveDriver,
 
       let driver = res.get()
 
-      try:
-        # The table should exist beforehand.
-        let newTableRes = waitFor driver.createMessageTable()
-        if newTableRes.isErr():
-          return err("error creating table: " & newTableRes.error)
-      except CatchableError:
-        return err("exception creating table: " & getCurrentExceptionMsg())
+      # Database migration
+      if migrate:
+        let migrateRes = await archive_postgres_driver_migrations.migrate(driver)
```

Comment:
Is it necessary to have postgres driver's migrate async while the prototype and sqlLite version is sync?

Reply:
See https://github.com/waku-org/nwaku/pull/2477#discussion_r1508264944